src/core/Microsoft.PowerToys.Settings.UI.Lib/PowerRenameSettings.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+
+namespace Microsoft.PowerToys.Settings.UI.Lib
+{
+    public class PowerRenameSettings : BasePTModuleSettings
+    {
+        public PowerRenameProperties properties { get; set; }
+
+        public PowerRenameSettings()
+        {
+            this.properties = new PowerRenameProperties();
+            this.version = "1";
+            this.name = "_unset_";
+        }
+
+        public PowerRenameSettings(string ptName)
+        {
+            this.properties = new PowerRenameProperties();
+            this.version = "1";
+            this.name = ptName;
+        }
+
+        public override string ToString()
```

Comment:
Overriding `ToString` method for json serialization feelks like doing 2 things at once. Maybe introduce `ToJson` method?

Reply:
I agree, let's make this explicit `ToJsonString()`

Reply:
resolved here: https://github.com/microsoft/PowerToys/pull/1813/commits/9c73fe398214544a36fe6a6e2f0e91c814abf92b